dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -14,7 +14,7 @@ ipv6 prefix-list PL_LoopbackV6 permit {{ get_ipv6_loopback_address(LOOPBACK_INTE
 {% endif %}
 !
 !
-{% if DEVICE_METADATA['localhost']['sub_role'] == 'FrontEnd' %}
+{% if multi_npu() %}
```

Comment:
do we intend to use multi_npu or multi_asic? I thought @rlhui plan to use multi_asic throughout the project?

Reply:
Changed to multi_asic in the latest commit.